### PR TITLE
GDS Admin API: Adds Get ReviewToken endpoint

### PR DESCRIPTION
**Pull request by bbengfort**
Adds a `GET /v2/vasps/:id/review endpoint` for fetching the admin verification token if it's available. Used in the UI to determine if a VASP can be reviewed or not.

Fixes SC-2763

### CODE REVIEW

File: pkg/gds/admin.go [hunk 1]
```diff
@@ -1481,6 +1482,52 @@ func (s *Admin) DeleteReviewNote(c *gin.Context) {
...
+	// Construct the reply
+	out = &admin.ReviewTokenReply{}
+	if out.AdminVerificationToken, err = models.GetAdminVerificationToken(vasp); err != nil {
+		log.Error().Err(err).Str("id", vaspID).Msg("could not retrieve admin verification token")
+		c.JSON(http.StatusInternalServerError, admin.ErrorResponse("could not retrieve admin verification token"))
+		return
+	}
+
+	// Check that an admin verification token will be returned
+	if out.AdminVerificationToken == "" {
```
Comment (pdeziel) [inline pkg/gds/admin.go:1521]: Nice check!

File: pkg/gds/admin_test.go [hunk 2]
```diff
@@ -943,6 +943,56 @@ func (s *gdsTestSuite) TestDeleteReviewNote() {
...
+func (s *gdsTestSuite) TestReviewToken() {
+	s.LoadFullFixtures()
+
+	require := s.Require()
+	a := s.svc.GetAdmin()
+
+	echo := s.fixtures[vasps]["echo"].(*pb.VASP)
+	juliet := s.fixtures[vasps]["juliet"].(*pb.VASP)
+
+	require.NotEqual(pb.VerificationState_PENDING_REVIEW, echo.VerificationStatus, "charlie must not be in PENDING_REVIEW for this test to pass")
```
Comment (pdeziel) [inline pkg/gds/admin_test.go:956]:
```suggestion
	require.NotEqual(pb.VerificationState_PENDING_REVIEW, echo.VerificationStatus, "echo must not be in PENDING_REVIEW for this test to pass")
```

File: pkg/gds/admin_test.go [hunk 3]
```diff
@@ -943,6 +943,56 @@ func (s *gdsTestSuite) TestDeleteReviewNote() {
...
+	require.Equal(pb.VerificationState_PENDING_REVIEW, juliet.VerificationStatus, "juliet must be in PENDING_REVIEW for this test to pass")
```
Comment (pdeziel) [inline pkg/gds/admin_test.go:957]: Thanks for adding these checks to help avoid future confusion!

**Review by pdeziel — APPROVED** [review]
Looks good to me!

**Comment by bbengfort** [comment]
Thanks for catching echo vs. charlie!

**Pull request merged**